neo/rawio/baserawio.py

```diff
@@ -111,22 +124,22 @@ class BaseRawIO:
     description = ''
     extensions = []
 
-    rawmode = None  # one key in possible_raw_modes
+    rawmode = None  # one key from possible_raw_modes
 
     def __init__(self, use_cache=False, cache_path='same_as_resource', **kargs):
         """
+        :TODO: Why multi-file would have a single filename is confusing here - shouldn't
+        the name of this argument be filenames_list or filenames_base or similar?
```

Comment:
Lets see for new IOs that really need list. Most of then need one of the file and the other ones can be induced.